libs/ledgerjs/packages/hw-app-btc/README.md

```diff
@@ -232,7 +232,7 @@ Returns **[Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/
 
 #### signP2SHTransaction
 
-To obtain the signature of multisignature (P2SH) inputs, call signP2SHTransaction_async with the folowing parameters
```

Comment:
yes, you are right. 
but you should change also https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledgerjs/packages/hw-app-btc/src/Btc.ts#L193
the doc here is generated from this file